_includes/configuration/icon_list.md

```diff
@@ -117,6 +119,8 @@
 | `selected` | ![selected.svg]({{ site.baseurl }}/images/icons/selected.svg) | `selected.svg` | 
 | `settings` | ![settings.svg]({{ site.baseurl }}/images/icons/settings.svg) | `settings.svg` | 
 | `sharpen` | ![sharpen.svg]({{ site.baseurl }}/images/icons/sharpen.svg) | `sharpen.svg` | 
+| `sort-asc` | ![sort-asc.svg]({{ site.baseurl }}/images/icons/sort-asc.svg) | `sort-asc.svg` | 
```

Comment:
Should we be including this, given they appear to be broken (backwards) at the moment?

Reply:
not sure @EkimChau ?

Reply:
I left a comment on [AT-11](https://ephocks.atlassian.net/browse/AT-11). An internal survey said point upwards. period. Google image search says 50/50. 🤷‍♂️

Reply:
Hi all,
Given that they are not being used in the product at this time, I've removed these icons from the docs while this gets sorted out/discussed. They can be re-included at any time.